API documentation

bibaev opened the pull request:
Resolve #4

bibaev committed:
add examples and signature for each request

bibaev committed:
add types description

bibaev committed:
add markdown API description.

bbugaev committed:
Example of API Blueprint usage

Tranlated first part of API description from Markdown
to API Blueprint

bibaev committed:
fix tiny error

bibaev committed:
api description fixed

bbugaev committed:
Yet another login method

bbugaev commented on apiary.apib:
```diff
@@ -1,42 +1,263 @@
+    + Поля ответа:
+        + `api_version` `(string)` - Версия API
+        + `answer` `(object)` - Ответ сервера
+          + `token` `(string)` - Секрытный токет от сервера, используемый для дальнейшей авторизации.
```
Опечатки: секретный токен

bbugaev commented on apiary.apib:
```diff
@@ -1,42 +1,263 @@
+    + uid: `2133` (required, string) - Идентификатор пользователя.
+    + token: `12343534` (required, string) - Токен аутентификации.
+    + type: `1` (required, number) - Тип задач.
+    + count: `10` (optional, number) - Количество запрашиваевых задач.
```
Опечатка: запрашиваемых

bbugaev committed:
Minor fixes

* typos
* whitespaces
* formatting
* typographics

bibaev committed:
fixes in api description

    - add point in methods and parameters description
    - fix content types
    - change HTTP methods
